java/README.md

```diff
@@ -1,34 +1,61 @@
-# Getting Started with the Crate.IO JDBC Sample App
-This Crate JDBC backend uses [Spark][1] as web framework and the Crate [JDBC driver][2].
+# Getting Started with the Crate JDBC Sample Application
+This sample application uses [Spark][1] as web framework and the
+[PostgreSQL JDBC driver][2] to access the Crate database.
 
 ## Requirements
+- [Spark Framework][1]
+- [PostgreSQL JDBC][2]
 - [Java 8][3]
 
 ## Usage
-To build and run the sample application we use the [Apache Maven][4] build tool.
+To build and run the sample application we use the [Apache Maven][4]
+build tool.
+
+### Run Crate Server
+Get the latest Crate [nightly build][5] and start the Crate from the
```

Comment:
This demo requires the experimental Postgres Protocol for Crate which will be releases with the `0.56.0`. Until there is no such release, you need to us the latest nightly build ....